README.md

```diff
@@ -28,24 +28,22 @@ If you want to change the look and feel of the ownCloud documentation, you can f
 
 ## How To Generate PDFs
 
-To generate a PDF version of one or more of the manuals, you need to have `asciidoctor-pdf` installed.
-With it installed, run the command below, in the root docs directory, to generate a PDF version of a manual.
-The PDF file will be generated in the same directory as where the command is run, and will be named after the configuration file, with a `.pdf` extension.
+To generate the PDF copies of the three manuals (administration, developer, and user), you need to have `asciidoctor-pdf`, and GNU make installed installed.
+If they’re installed, run the command below in the root directory, to generate the manuals.
```

Comment:
``in the root directory`` of what?

Reply:
I'm currently experimenting with using the ROOT module as an introductory/overview module, and renaming the current ROOT module to be the "administration manual". Based on that, I'll make changes here.